lib/utils/args.js

```diff
@@ -54,21 +54,24 @@ function _normaliseFlagsAndArs(argsIntoCommand, cmd) {
     // bail on old style commands
     return argsIntoCommand;
   }
-  var processArgs = _.clone(process.argv);
+
+  const normalisedArgs = _.clone(argsIntoCommand);
```

Comment:
Need to use the passed arguments here, not `process.argv` since `process.argv` only works if using as  CLI. This new method works for both CLI and as a module.